packages/gestalt/src/moduleUtils.js

```diff
@@ -0,0 +1,40 @@
+// @flow strict
+import React, { type Node } from 'react';
+import Box from './Box.js';
+import Icon from './Icon.js';
+import Text from './Text.js';
+
+// eslint-disable-next-line import/prefer-default-export
+export function renderModuleTitle(
```

Comment:
This isn't really a util function — it's a component. Feel free to move it to something like ModuleTitle.js, and we'll consider options for organizing our file tree in the future as it gets more cluttered.